packages/terra-folder-tree/src/subcomponents/FolderTreeItem.module.scss

```diff
@@ -14,8 +14,24 @@
     width: 100%;
   
     &:hover {
-      background-color: var(--terra-folder-tree-item-hover-background-color, #f4fafe);
       cursor: pointer;
+
+      &:not(.hover-within) {
+        background-color: var(--terra-folder-tree-item-hover-background-color,  rgb(203, 230, 247, 0.5));
+      }      
+    }
+
+    &.hover-within {
+      .radio-target::before {
+        content: '';
+        position: absolute;
+        top: 0;
+        right: 0;
+        left: 0;
+        bottom: 0;
+        background-color: var(--terra-folder-tree-item-hover-background-color, rgb(203, 230, 247, 0.5));
+        border-radius: 50%;
```

Comment:
The pseudo class is used to create a circle on hover, as the actual click area needs to stay square.